src/screenshot/screenshot-compare.ts

```diff
@@ -146,6 +146,7 @@ async function getMismatchedPixels(pixelmatchModulePath: string, pixelMatchInput
     }, timeout);
 
     try {
+      let error: string | undefined;
```

Comment:
Does anything ever assign a value to this? Unsure if I'm missing a place where we pass a reference to this variable or assign it in a callback, but it doesn't appear to be used anywhere other than in [this error message](https://github.com/ionic-team/stencil/pull/5167/files#diff-5eb153c1602d2d29da2e3808720879c3de746c2c68e057831c38fb378fc5833fR177)

Reply:
Good catch, I missed to copy the piece of code that would capture the error message from `stderr`.